### PR TITLE
Fix PEtab import for performance test model

### PR DESCRIPTION
**Pull request by dweindl**

### CODE REVIEW

**Comment by FFroehlich** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- python/amici/petab_objective.py  6
- python/amici/petab_import.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/ICB-DCM/AMICI/pullRequest?prid=4971370&bid=16422085)

**Comment by FFroehlich** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- python/amici/petab_objective.py  6
- python/amici/petab_import.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/ICB-DCM/AMICI/pullRequest?prid=4971370&bid=16422085)

File: python/amici/petab_import.py [hunk 1]
```diff
@@ -119,11 +120,12 @@ def get_fixed_parameters(
                                   f"initial assignment for {compartments}")
 
     species = [col for col in condition_df
-               if sbml_model.getSpecies(col) is not None]
+               if not np.issubdtype(condition_df[col].dtype, np.number)
```
Comment (yannikschaelte) [inline python/amici/petab_import.py]: why not jut ``issubclass(condition_df[col].dtype, numbers/np.number)``?

File: python/amici/petab_objective.py [hunk 2]
```diff
@@ -407,9 +407,38 @@ def _get_par(model_par, value):
...
+                condition_id = condition[PREEQUILIBRATION_CONDITION_ID]
+            else:
+                condition_id = condition[SIMULATION_CONDITION_ID]
+
+            x0[species_idx] = petab_problem.condition_df.loc(condition_id,
+                                                             species_id)
+
+        edata.x0 = x0
+
+    # TODO: implement: In case of parametric overrides, they would have be
```
Comment (yannikschaelte) [inline python/amici/petab_objective.py]:
```suggestion
    # TODO: implement: In case of parametric overrides, they would have to be
```
and link issue?

**Comment by FFroehlich** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- python/amici/petab_objective.py  6
- python/amici/petab_import.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/ICB-DCM/AMICI/pullRequest?prid=4971370&bid=16422085)

File: python/amici/petab_import.py (hunk 1 above)
Comment (dweindl) [reply python/amici/petab_import.py]: In some other situation `issubdtype` worked whereas `issubclass` did not. Here it seems the latter is also fine.

Comment (yannikschaelte) [reply python/amici/petab_import.py]: both fine for me, was just asking whether the latter had not worked for some reason (numbers subtypes with built-ins and numpy versions can be picky in python ime)

**Comment by FFroehlich** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- python/amici/petab_objective.py  6
- python/amici/petab_import.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/ICB-DCM/AMICI/pullRequest?prid=4971370&bid=16422085)

File: python/amici/petab_import.py (hunk 1 above)
Comment (dweindl) [reply python/amici/petab_import.py]: That was why :D

```
  File "/home/runner/.local/lib/python3.6/site-packages/amici/petab_import.py", line 124, in get_fixed_parameters
    species = [col for col in condition_df
  File "/home/runner/.local/lib/python3.6/site-packages/amici/petab_import.py", line 125, in <listcomp>
    if not issubclass(condition_df[col].dtype, Number)
  File "/usr/lib/python3.6/abc.py", line 223, in __subclasscheck__
    if issubclass(subclass, rcls):
TypeError: issubclass() arg 1 must be a class
Command exited with non-zero status 1
```

**Comment by FFroehlich** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- python/amici/petab_objective.py  6
- python/amici/petab_import.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/ICB-DCM/AMICI/pullRequest?prid=4971370&bid=16422085)

**Comment by FFroehlich** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- python/amici/petab_objective.py  6
- python/amici/petab_import.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/ICB-DCM/AMICI/pullRequest?prid=4971370&bid=16422085)

**Comment by FFroehlich** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- python/amici/petab_objective.py  6
- python/amici/petab_import.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/ICB-DCM/AMICI/pullRequest?prid=4971370&bid=16422085)

**Comment by FFroehlich** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- python/amici/petab_objective.py  6
- python/amici/petab_import.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/ICB-DCM/AMICI/pullRequest?prid=4971370&bid=16422085)

**Comment by FFroehlich** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- python/amici/petab_objective.py  6
- python/amici/petab_import.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/ICB-DCM/AMICI/pullRequest?prid=4971370&bid=16422085)

**Comment by FFroehlich** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- python/amici/petab_objective.py  6
- python/amici/petab_import.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/ICB-DCM/AMICI/pullRequest?prid=4971370&bid=16422085)

**Comment by FFroehlich** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- python/amici/petab_objective.py  6
- python/amici/petab_import.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/ICB-DCM/AMICI/pullRequest?prid=4971370&bid=16422085)

**Comment by FFroehlich** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- python/amici/petab_objective.py  6
- python/amici/petab_import.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/ICB-DCM/AMICI/pullRequest?prid=4971370&bid=16422085)

**Comment by FFroehlich** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- python/amici/petab_objective.py  6
- python/amici/petab_import.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/ICB-DCM/AMICI/pullRequest?prid=4971370&bid=16422085)

**Comment by FFroehlich** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- python/amici/petab_objective.py  6
- python/amici/petab_import.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/ICB-DCM/AMICI/pullRequest?prid=4971370&bid=16422085)

**Pull request merged**